connector.go

```diff
@@ -136,6 +136,19 @@ func (c *connector) Connect(ctx context.Context) (driver.Conn, error) {
 		return nil, err
 	}
 
+	res, err := mc.Query("SELECT CONNECTION_ID()", nil)
```

Comment:
Do not use query. Use connection id in handshake packet instead.

Reply:
See here.
https://github.com/go-sql-driver/mysql/blob/4591e42e65cf483147a7c7a4f4cfeac81b21c917/packets.go#L210-L212

Reply:
Done